src/commands/jira/askToCreateNewTicket.ts

```diff
@@ -28,14 +29,43 @@ export const askToCreateNewTicket = async (): Promise<Ticket> => {
       type: "text",
       name: "description",
       message: "Add a description:"
+    },
+    {
+      type: "text",
+      name: "aliases",
+      message: "Add comma-separated aliases for easy searching.",
+      validate: async (input: string) => {
+        if (input == null || input.trim() === "") {
+          return true;
+        }
```

Comment:
Wouldn't we want to return false here?

Reply:
In this case, no - we return `true` because an empty input "passes" validation, since this field is optional.